README.rst

```diff
@@ -7,47 +7,160 @@ BitShuffle README
+"let me show you this cool shell script I wrote", or "here look at this 10 line
+long log file".
 
-Why Not Use PasteBin/HasteBin/Sprunge/Etc
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Why Not Use PasteBin/HasteBin/Sprunge/Etc?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 These services are designed specifically for transferring plain text data, and
-often mangle binary data. They usually have size limitations, and do not
-support multiple files.
+often mangle binary data. They usually have size limitation as well.
 
-CONTRIBUTING
----
+Is This Really Useful?
+~~~~~~~~~~~~~~~~~~~~~~
+
+The authors of BitShuffle find it useful. Maybe you will too. Maybe not.
+
+Why so Much CI / Testing?
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The amount of automated tests may seem high for a project as small as
+BitShuffle is. However, BitShuffle is intended to be a tool used on a daily
+basis (as it is by its authors), inside of pipelines, and possibly inside of
+other automation. It is critical thus that it not break or behave in strange or
+unusual ways for the same reason ``ls`` needs to not break on weird edge cases
+- it's used too frequently.
+
+Can I Embed BitShuffle in my Project?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, but please wait until we have a stable release. The data packet format may
+change without warning until there is at least one stable release.
+
+Does BitShuffle Have a Stable API?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Not at this time, but it will in the future as the project matures a bit. Until
+then use BitShuffle as a Python module at your own risk.
+
+Installation
+============
+
+Dependencies
+------------
+
+To install/run BitShuffle:
 
-The only thing needed to run the executable is python (2 or 3). However,
-the tests require `bc`, `uuidgen`, and `shasum`. On debian-based systems, these
-can be installed with `sudo apt-get install bc uuid-runtime coreutils`.
+* POSIX-ish operating system, or Windows (as of #38).
 
-BitShuffel Data Packet Specification
-====================================
+* Python (>= 2.7)
+
+To run BitShuffle's automated tests locally:
+
+* POSIX ``sh`` compliant shell interpreter
+* ``uuidgen``
+* ``travis`` (hint: ``gem install travis``)
+* ``bc``
+* ``/tmp`` must exist and be write-able
+* ``pycodestyle``
+
+Installing with ``setup.py``
+----------------------------
+
+Simply run ``python ./setup.py install``.
+(Note: this assumes ``which python`` is identical to ``python``)
+
+Installing Manually
+-------------------
+
+If you are only going to be using BitShuffle as a script, not as a python
+module, you can also just drop ``bitshuffle/bitshuffle.py`` into ``$PATH`` (I
+suggest symlinking to ``~/bin/bitshuffle``).
+
+Installing a Binary Release
+---------------------------
+
+This is not possible yet, but in the future, there will be static builds of
+BitShuffle that can be run standalone.
+See also `#11 <https://github.com/charlesdaniels/bitshuffle/issues/11>`_.
```

Comment:
This has been resolved, we should add a note of it as well as a link

Reply:
Fixed in 
917686c